src/db/User.js

```diff
@@ -265,11 +287,11 @@ export default class User extends Model {
 
 /**
  * Get an IRC host safe version of a rat name for use in a virtual host
- * @param {Model} rat the rat which name should be used
+ * @param {string} rat the rat name which should be used
  * @returns {string} the generated irc safe name
  */
 function getIRCSafeName (rat) {
-  let ratName = rat.name
+  let ratName = rat
```

Comment:
nitpicking, but given what "rat" means elsewhere within this codebase, wouldn't it be better to name the variable `cmdrName` to make it clearer what the name is? :P

Reply:
Updated